conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -42,11 +45,13 @@ echo -e "\n\nMangling homebrew in the CI to avoid conflicts."
 echo -e "\n\nRunning the build setup script."
 {{ build_setup }}
 {%- endif %}
-endgroup "Configuring conda"
+
+{ set +x; endgroup "Configuring conda" } 2> /dev/null
 
 set -e
 
-startgroup "Running conda $BUILD_CMD"
+{ set +x; startgroup "Running conda $BUILD_CMD" } 2> /dev/null
```

Comment:
Can we remove this group and make conda-build command not be folded?

Reply:
Done. Check how it looks here:
https://github.com/jaimergp-feedstocks/test-feedstock/runs/2134595185?check_suite_focus=true